modules/nf-core/porechop/abi/main.nf

```diff
@@ -0,0 +1,36 @@
+process PORECHOP_ABI {
```

Comment:
I know when we chatted you mentioned we modified some of these. Make sure they go to the `nf-core-modified` folder with documentation as what was modified and why. Maybe it was `racon`

Reply:
Created `nf-core-modified` modules directory, the modified ones were racon and the minimap2/align modules. Comment in those modules what was changed and why